src/dsp_tools/commands/xmlupload/make_rdf_graph/make_resource_and_values.py

```diff
@@ -27,17 +31,37 @@ def create_resource_with_values(
     Returns:
         A resource serialised in json-ld type format.
     """
-    res_bnode = BNode()
-    res = _make_resource(resource=resource, bitstream_information=bitstream_information, lookup=lookup)
-    vals = make_values(resource, res_bnode, lookup)
-    res.update(vals)
+
+    res = _make_resource(resource=resource, lookup=lookup)
+    res.update(_make_values_dict_from_resource(resource, bitstream_information, lookup))
     res.update(lookup.jsonld_context.serialise())
     return res
 
 
-def _make_resource(
+def _make_values_dict_from_resource(
     resource: XMLResource, bitstream_information: BitstreamInfo | None, lookup: Lookups
 ) -> dict[str, Any]:
+    res_bnode = BNode()
+
+    properties_graph, last_prop_name = make_values(resource.properties, resource.restype, res_bnode, lookup)
+
+    if resource.iiif_uri:
```

Comment:
This is what we discussed this morning. It is a lot of logic here (resolve permissions, etc.) which in the best case would happen previously. But until we have the anti-corruption layer, I prefer to keep it here. Rather than hand over the lookups into the `make_iiif_uri_value_graph` function.